test/runcitests.cmd

```diff
@@ -127,10 +130,13 @@ set _HadFailures=0
 
   call :do %_TestDir%\runnativetests.cmd -%1%2 > %_TestDir%\logs\%1_%2\nativetests.log 2>&1
 
-  if ERRORLEVEL 1 set _HadFailures=1
+  if %ERRORLEVEL% NEQ 0 (
+    echo -- runcitests.cmd ^>^> runnativetests.cmd failed

```

Comment:
Oh I guess it actually does contain information that isn't echoed to stdout. Maybe we can echo the log file to stdout to solve this problem?